tests/fsharp/core/array/test.fsx

```diff
@@ -1368,6 +1368,28 @@ module bug872632 =
 
     do check "bug872632" Foo.x.Length 8
 
+module CheckUnionTypesAreSealed =
+    open System
+    do check "vwllfewlkefw1" (typedefof<list<int>>.IsSealed) true
```

Comment:
`IsSealed` is not available in profiles 7, 78, or 259, which causes the tests in tests\fsharp\core\netcore to fail here